kuma/javascript/src/active-banner.jsx

```diff
@@ -189,9 +189,28 @@ function Banner(props: BannerProps) {
     );
 }
 
+export const MDN_BROWSER_COMPAT_REPORT_ID = 'mdn_browser_compat_report';
 export const DEVELOPER_NEEDS_ID = 'developer_needs';
 export const SUBSCRIPTION_ID = 'subscription_banner';
 
+function MDNBrowserCompatReportBanner() {
+    return (
+        <Banner
+            id={MDN_BROWSER_COMPAT_REPORT_ID}
+            classname="developer-needs"
+            title={gettext('MDN Browser Compatibility Report')}
+            copy={gettext(
+                'A deep dive into web compatibility frustrations, with useful insights into how they could be overcome.'
+            )}
+            cta={gettext('Read the report')}
```

Comment:
```suggestion
            cta={gettext('Read the report (PDF, 1.8mb)')}
```

As per @robnyman 's suggestion.